sdk/policyinsights/azure-mgmt-policyinsights/azure/mgmt/policyinsights/models/_models_py3.py

```diff
@@ -1637,27 +1729,33 @@ class PolicyEvaluationResult(_serialization.Model):
     :vartype evaluation_result: str
     :ivar evaluation_details: The detailed results of the policy expressions and values that were
      evaluated.
-    :vartype evaluation_details: ~azure.mgmt.policyinsights.models.PolicyEvaluationDetails
+    :vartype evaluation_details:
+     ~azure.mgmt.policyinsights.models.CheckRestrictionEvaluationDetails
+    :ivar effect_details: The details of the effect that was applied to the resource.
+    :vartype effect_details: ~azure.mgmt.policyinsights.models.PolicyEffectDetails
     """
 
     _validation = {
         "policy_info": {"readonly": True},
         "evaluation_result": {"readonly": True},
         "evaluation_details": {"readonly": True},
+        "effect_details": {"readonly": True},
     }
 
     _attribute_map = {
         "policy_info": {"key": "policyInfo", "type": "PolicyReference"},
         "evaluation_result": {"key": "evaluationResult", "type": "str"},
-        "evaluation_details": {"key": "evaluationDetails", "type": "PolicyEvaluationDetails"},
+        "evaluation_details": {"key": "evaluationDetails", "type": "CheckRestrictionEvaluationDetails"},
+        "effect_details": {"key": "effectDetails", "type": "PolicyEffectDetails"},
     }
 
-    def __init__(self, **kwargs):
+    def __init__(self, **kwargs: Any) -> None:
```

Comment:
For the readonly property, it will not be displayed in signatures which can't be dectected by breaking change detector tool.